server/src/models/Game.ts

```diff
@@ -0,0 +1,45 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
```

Comment:
Can we fix this lint error before merge or temporary suspend it to enable our CI to run tests
```suggestion
/* eslint-disable no-unused-vars */
```